main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func xenDit(i int) string {
+	//For every multiplication of 3 and the number of occurrences is an odd number, print “Xen” instead.
+	//If it’s a multiplication of 3 and the number of occurrences is an even number, print “dit”.
+	//And for every multiplication of 10, print “Xendit”
+	// TODO: write down the logic with if statement
+	if i%3 == 0 {
+		if i%2 != 0 {
+			return "Xen"
+		} //even
+		if i%2 == 0 && i%10 != 0 {
+			return "Dit"
+		}
+	}
+	if i%10 == 0 {
+		return "Xendit"
+	}
```

Comment:
woaa - it's my 1st time to see someone use different pattern to solve this problem. Rawrsome! :star_struck: